snuba/query/parser/expressions.py

```diff
@@ -101,65 +123,51 @@ def visit_low_pri_arithmetic(
         self,
         node: Node,
         visited_children: Tuple[
-            Any, Expression, Any, Union[Node, Tuple[LowPriOperator, Expression]]
+            Any, Expression, Any, Union[Node, LowPriTuple, List[LowPriTuple]]
         ],
     ) -> Expression:
         _, term, _, exp = visited_children
 
-        if exp == "":
+        if isinstance(exp, Node):
             return term
 
+        if isinstance(exp, LowPriTuple):
+            if exp.op == LowPriOperator.PLUS:
+                return plus(term, exp.arithm)
+            elif exp.op == LowPriOperator.MINUS:
+                return minus(term, exp.arithm)
```

Comment:
This code is a duplication of the code at line 141-144. 
You can fit it into an inner function in this method and reuse it.

In python you can declare a function inside another one.
https://realpython.com/inner-functions-what-are-they-good-for/

Reply:
Implemented.